Model/EpsPaymentMethod.php

```diff
@@ -0,0 +1,152 @@
+     *
+     * @return Transaction
+     *
+     * @since 1.2.0
+     */
+    public function getTransaction()
+    {
+        return new EpsTransaction();
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     *
+     * @since 1.2.0
+     */
+    public function getConfigFields()
+    {
+        $aAdditionalFields = [
+            'descriptor' => [
+                'type'  => 'select',
+                'field' => 'oxpayments__wdoxidee_descriptor',
+                'options' => [
+                    '1' => Helper::translate('wd_yes'),
+                    '0' => Helper::translate('wd_no'),
+                ],
+                'title' => Helper::translate('wd_config_descriptor'),
+                'description' => Helper::translate('wd_config_descriptor_desc'),
+            ],
+            'additionalInfo' => [
+                'type'  => 'select',
+                'field' => 'oxpayments__wdoxidee_additional_info',
+                'options' => [
+                    '1' => Helper::translate('wd_yes'),
+                    '0' => Helper::translate('wd_no'),
+                ],
+                'title' => Helper::translate('wd_config_additional_info'),
+                'description' => Helper::translate('wd_config_additional_info_desc'),
+            ],
+            'deleteCanceledOrder' => [
+                'type' => 'select',
+                'field' => 'oxpayments__wdoxidee_delete_canceled_order',
+                'options' => [
+                    '1' => Helper::translate('wd_yes'),
+                    '0' => Helper::translate('wd_no'),
+                ],
+                'title' => Helper::translate('wd_config_delete_cancel_order'),
+                'description' => Helper::translate('wd_config_delete_cancel_order_desc'),
+            ],
+            'deleteFailedOrder' => [
+                'type' => 'select',
+                'field' => 'oxpayments__wdoxidee_delete_failed_order',
+                'options' => [
+                    '1' => Helper::translate('wd_yes'),
+                    '0' => Helper::translate('wd_no'),
+                ],
+                'title' => Helper::translate('wd_config_delete_failure_order'),
+                'description' => Helper::translate('wd_config_delete_failure_order_desc'),
+            ],
+        ];
+
+        return array_merge(parent::getConfigFields(), $aAdditionalFields);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     *
+     * @since 1.2.0
+     */
+    public function getPublicFieldNames()
+    {
+        return array_merge(parent::getPublicFieldNames(), [
+            'descriptor',
+            'additionalInfo',
+            'deleteCanceledOrder',
+            'deleteFailedOrder',
+        ]);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @param string $sAction
+     *
+     * @return SepaCreditTransferPaymentMethod
+     *
+     * @since 1.2.0
+     */
+    public function getPostProcessingPaymentMethod($sAction)
```

Comment:
This is no longer used. Extending from `SepaCreditTransferPaymentMethod` is how we do it now. See [GiropayPaymentMethod](https://github.com/wirecard/oxid-ee/blob/master/Model/GiropayPaymentMethod.php) for an example.

Reply:
:+1: